server/src/main/java/org/apache/druid/server/coordinator/CoordinatorCompactionConfig.java

```diff
@@ -34,11 +36,13 @@ public class CoordinatorCompactionConfig
   private static final double DEFAULT_COMPACTION_TASK_RATIO = 0.1;
   private static final int DEFAULT_MAX_COMPACTION_TASK_SLOTS = Integer.MAX_VALUE;
   private static final boolean DEFAULT_USE_AUTO_SCALE_SLOTS = false;
+  private static final CompactionEngine DEFAULT_COMPACTION_ENGINE = CompactionEngine.NATIVE;
```

Comment:
Cluster-level default should be a config and could be exposed as a separate API.
That API should validate all existing datasource compaction configs.

(this change maybe done in a follow up).

Reply:
Will expose the API in a follow-up PR. The cluster-level config has already been added.